### PR TITLE
Icons scss optimisation

### PR DESCRIPTION
**Pull request by skjnldsv**
Optimisation and simplification of the icons.scss file

@nextcloud/designers

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]
tested and works

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

File: core/css/icons.scss [hunk 1]
```diff
@@ -1,103 +1,88 @@
+/**
+ * @copyright Copyright (c) 2017, John Molakvoæ (skjnldsv@protonmail.com)
```
Comment (jancborchardt) [inline core/css/icons.scss]: @skjnldsv here too, the authors from the original file need to be included too so the copyright is correct. :)

Comment (skjnldsv) [inline core/css/icons.scss]: There wasn't any, should I get them from the blame?

Comment (MorrisJobke) [inline core/css/icons.scss]: > There wasn't any, should I get them from the blame?

yes

Comment (skjnldsv) [inline core/css/icons.scss]: All of them? ^^

Comment (MorrisJobke) [inline core/css/icons.scss]: Yes - try to get all of them in - beside the ones only touching 1-2 lines.

Comment (skjnldsv) [inline core/css/icons.scss]: I will do a copyright pr! :)

Comment (jancborchardt) [inline core/css/icons.scss]: @skjnldsv the majority was me as I created the original file in 2014 and edited most of it. ;) https://github.com/nextcloud/server/blame/stable11/core/css/icons.css

Comment (skjnldsv) [inline core/css/icons.scss]: Just a question, but why didn't you put you as a copyright back then? :p

Comment (jancborchardt) [inline core/css/icons.scss]: @skjnldsv good question, too busy writing the actual code I guess. ;) That doesn’t change the fact though it’s under copyright

Comment (skjnldsv) [inline core/css/icons.scss]: Of course! 😉

**Pull request merged**